lib/Chat/ChatManager.php

```diff
@@ -161,7 +166,12 @@ public function sendMessage(Room $chat, Participant $participant, string $actorT
 			// Update last_message
 			$chat->setLastMessage($comment);
 
-			$mentionedUsers = $this->notifier->notifyMentionedUsers($chat, $comment);
+			$mentionedUsers = [];
+			if ($replyTo instanceof IComment) {
+				$mentionedUsers = $this->notifier->notifyReplyToAuthor($chat, $comment, $replyTo);
```

Comment:
For consistency with the comment in `notifyMentionedUsers()` this should be renamed too to `$alreadyNotifiedUsers` or something like that.